packages/fluentui/docs/src/examples/components/Tree/Usage/TreeMultiselectExample.shorthand.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { Tree, CustomSelectIndicatorProps, Flex } from '@fluentui/react-northstar';
+
+const customCheckbox = ({ selected, selectable, selectableParent, expanded, ...props }: CustomSelectIndicatorProps) => {
+  return (
+    <>
+      <Flex {...props}>{selectableParent && expanded && 'custom select all'}</Flex>
+      {selectable && <input data-is-focusable={false} type="checkbox" readOnly checked={selected} {...props} />}
```

Comment:
Are you sure it is correct to spread rest props on both the `Flex` and the `input`?

Reply:
fixed